api/src/main/openapi/components/schemas/portfolio-metrics-response.yaml

```diff
@@ -0,0 +1,123 @@
+# This file is part of Dependency-Track.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OWASP Foundation. All Rights Reserved.
+type: object
+properties:
+  critical:
+    type: integer
+    format: int32
+  high:
+    type: integer
+    format: int32
+  medium:
+    type: integer
+    format: int32
+  low:
+    type: integer
+    format: int32
+  unassigned:
+    type: integer
+    format: int32
+  vulnerabilities:
+    type: integer
+    format: int32
+  projects:
+    type: integer
+    format: int32
+  vulnerable_projects:
+    type: integer
+    format: int32
+  components:
+    type: integer
+    format: int32
+  vulnerable_components:
+    type: integer
+    format: int32
+  suppressed:
+    type: integer
+    format: int32
+  findings_total:
+    type: integer
+    format: int32
+  findings_audited:
+    type: integer
+    format: int32
+  findings_unaudited:
+    type: integer
+    format: int32
+  inherited_risk_score:
+    type: number
+    format: double
+  policy_violations_fail:
+    type: integer
+    format: int32
+  policy_violations_warn:
+    type: integer
+    format: int32
+  policy_violations_info:
+    type: integer
+    format: int32
+  policy_violations_total:
+    type: integer
+    format: int32
+  policy_violations_audited:
+    type: integer
+    format: int32
+  policy_violations_unaudited:
+    type: integer
+    format: int32
+  policy_violations_security_total:
+    type: integer
+    format: int32
+  policy_violations_security_audited:
+    type: integer
+    format: int32
+  policy_violations_security_unaudited:
+    type: integer
+    format: int32
+  policy_violations_license_total:
+    type: integer
+    format: int32
+  policy_violations_license_audited:
+    type: integer
+    format: int32
+  policy_violations_license_unaudited:
+    type: integer
+    format: int32
+  policy_violations_operational_total:
+    type: integer
+    format: int32
+  policy_violations_operational_audited:
+    type: integer
+    format: int32
+  policy_violations_operational_unaudited:
+    type: integer
+    format: int32
+  first_occurrence:
+    type: string
+    format: date-time
+    example: "2025-07-07T15:44:10.377Z"
+  last_occurrence:
+    type: string
+    format: date-time
+    example: "2025-07-07T15:44:10.377Z"
```

Comment:
We don't really use this distinction internally anymore (we only look at and partition by `lastOccurrence`). For the v2 API, let's use a single `timestamp` field to avoid confusion in the future.